tools/hive/hadoop-hive/templates/yarn-site.xml.template

```diff
@@ -0,0 +1,2 @@
+<configuration>
+</configuration>
```

Comment:
Is this file necessary? It's essentially empty.

Reply:
Strictly speaking, it's not necessary, but it serves as a placeholder for the "generate_configs.sh", so that the script can treat all config files (including `yarn-site`) equally.